jetty-ee8/jetty-ee8-demos/jetty-ee8-demo-jaas-webapp/pom.xml

```diff
@@ -26,14 +26,14 @@
               <!-- This is for convenience so that the src/etc/login.conf file can stay unmodified when copied to $jetty.home/etc directory -->
                 <jetty.base>${basedir}/src/main/config/modules/demo.d</jetty.base>
               <!-- Mandatory. This system property tells JAAS where to find the login module configuration file -->
-                <java.security.auth.login.config>${basedir}/src/main/config/modules/demo.d/ee8-demo-login.conf</java.security.auth.login.config>
```

Comment:
I understand ee8 demos are derived from ee9 demos with some Maven magic.
As such we should just change the ee9 files, not the ee8 ones?

Reply:
demos cannot be derived from ee9 demos. 
they are including some very differents jars etc... 
so I couldn;t do it for demos.